src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+$start-days:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sut 5,
+  sun 6;
+$cell-size: 100px;
+$gap: 1px;
+$full-width: 720px;
+$cell-color: #eee;
+$hovered-color: pink;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  padding: 20px 10px;
+  width: $full-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)
when all elements have the same `box-sizing` value, their behavior becomes more predictable